src/highlight.js

```diff
@@ -0,0 +1,18 @@
+const cards = document.querySelectorAll(".number-card")
+let randomNumbersInSequence = randomNumbersHolder.sort((function(a, b){return a-b}))
+let correctNumbersHolder = []
+
+cards.forEach(function (card) {
+  card.addEventListener("click", (e) => {
+    if (e.target.textContent == Math.min(...randomNumbersInSequence)) {
+      e.target.style.backgroundColor = "#686868"
```

Comment:
better to do this with a class, than changing inline css

Reply:
created new class called .highlighted-number-card